workflow-templates/daily-security-check.yml

```diff
@@ -0,0 +1,69 @@
+---
+name: Daily security check
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+  push:
```

Comment:
Is this intentional, to also run the GHA on `push`? If so, maybe update the GHA template description to include this fact?

Reply:
No, it should not run on push. I'll change that